packages/web-twig/src/Resources/components/ButtonLink/stories/ButtonLinkDisabled.twig

```diff
@@ -1,22 +1,18 @@
-<section class="docs-Section">
-    <h2 class="docs-Heading">Disabled</h2>
-
-    <Grid cols="1" tablet="2" desktop="3">
```

Comment:
I'm going to fix little things like this (grid settings, title case etc.) so we have an example to follow in the upcoming docs-unifying issues.